gutenberg.php

```diff
@@ -102,7 +102,7 @@ function gutenberg_menu() {
  * @since 9.4.0
  */
 function gutenberg_site_editor_menu() {
-	if ( gutenberg_is_fse_theme() ) {
+	if ( gutenberg_is_fse_enabled() ) {
```

Comment:
Allow showing the site-editor if the user has opted-in, or we're in an FSE theme.